client/src/app/(onboarding)/login/page.tsx

```diff
@@ -1,3 +1,195 @@
+"use client";
+import "./login.css";
+import { Button, Typography, Divider, TextField, Link } from "@mui/material";
+import { useState } from "react";
+
 export default function Page() {
-  return <h1>Login Placeholder</h1>;
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(false);
+  const [emailHelperText, setEmailHelperText] = useState("");
+  const [passwordHelperText, setPasswordHelperText] = useState("");
+
+  // Dummy Function, will handle login logic
+  const handleSubmit = () => {
+    console.log("Email:", email);
+    console.log("Password:", password);
+  };
+
+  const validateInput = () => {
+    setError(false);
+    setEmailHelperText("");
+    setPasswordHelperText("");
+
+    let hasError = false;
+
+    if (email === "") {
+      setError(true);
+      setEmailHelperText("Email is required");
+      hasError = true;
+    }
+
+    if (password === "") {
+      setError(true);
+      setPasswordHelperText("Password is required");
+      hasError = true;
+    }
+
+    if (hasError === false) {
+      handleSubmit();
+    }
+  };
+
+  return (
+    <div className="login-container">
+      <Typography variant="h2" sx={{ marginBottom: "10px" }}>
+        Welcome Back
+      </Typography>
+      <Button
+        className="border-line"
+        variant="outlined"
+        startIcon={
+          <img src="https://www.google.com/favicon.ico" alt="Google" />
+        }
+        sx={{
+          backgroundColor: "#ffff",
+          width: "708px",
+          borderRadius: "10px",
+          border: "4px solid #E0E4F2",
+        }}
+      >
+        Login with Google
+      </Button>
+
+      <div className="login-divider">
+        <Divider
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#E0E4F2",
+            marginRight: 2,
+          }}
+        />
+        <Typography
+          variant="h4"
+          sx={{
+            fontWeight: "bold",
+            fontSize: "1.25rem",
+            textAlign: "center",
+          }}
+        >
+          Or
+        </Typography>
+        <Divider
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#E0E4F2",
+            marginLeft: 2,
+          }}
+        />
+      </div>
+
+      <div className="login-box">
+        <div className="login-fields">
+          <Typography
+            variant="h5"
+            sx={{
+              textAlign: "start",
+              width: "500px",
+              marginBottom: "10px",
+            }}
+          >
+            Email
+          </Typography>
+          <TextField
```

Comment:
The TextField from the SignUp page is inconsistent with this TextField.

![image](https://github.com/user-attachments/assets/0899c1ff-e134-42e9-a955-ad8f5d5ebf41)